### ISSUE TITLE
Create Custom Mail Alias for  default domain

### ISSUE THREAD

**Issue by xanthos84** [anonaddy/anonaddy#632]
Hi,

why is it not possible to add a new custom mail alias for the default domain?
I also cant add the default domain to the "Domain" tab, because I get the error "is a local domain".

Thank you for clarifiying - as I need to create some custom mail aliases.

**Comment by dec-mail** [comment]
I would like to know too, as i will likely want to do this.

**Comment by dec-mail** [comment]
Hmm. 
TWO WEEKS that was Posted and NO Response at all (from anyone, let alone Addy Support!)

In that case, Addy might be all hype and no Support ... in which case i should prob select a different app for this purpose. Plenty of others, but would still be quite disappointing after spending so much time evaluating and reviewing so many apps.

And, together with the case of my separately posted fatal Addy error (no Verification email) before i can even start, it is not looking good.

**Comment by willbrowningme** [comment]
@dec-mail This user is requesting a feature for self-hosted instances that is not possible on the hosted (addy.io) instance.

On the hosted addy.io instance you can create custom aliases for your own domain without any issues.

@xanthos84 If you have:

```
# This allows you to receive emails as a catch-all at the apex domain e.g. *@example.com
ANONADDY_ADMIN_USERNAME=username
```

set in your `.env` file where `username` is your account's username, then you can create custom aliases on the fly for the default domain.

**Comment by xanthos84** [comment]
@willbrowningme thank you for your answer.
I got that line in my .env file, with my username filled in.
but i still cant create a new alias:
<img width="494" alt="image" src="https://github.com/anonaddy/anonaddy/assets/18182913/b11ec882-fd7a-406c-a708-e60c70fc6feb">

**Comment by xanthos84** [comment]
to clarify: i am using addy since about 2 years now selfhosted... and with the old versions before version 1.0, it was easily possible to create new custom aliases for my domain.
but now since 2 weeks I updated to v1.1.0 its not working anymore.

**Comment by willbrowningme** [comment]
Adding that to your `.env` does not allow you to create the aliases manually for the default domain (I need to implement that still).

But it does enable catch-all for your default domain so you don't need to create the alias beforehand.

**Comment by willbrowningme** [comment]
Will sort this shortly to allow manual creation of custom aliases at the default domains.

**Comment by xanthos84** [comment]
thank you so much!

**Comment by willbrowningme** [comment]
Fixed in [v1.2.0](https://github.com/anonaddy/anonaddy/releases/tag/v1.2.0).
